recipe/bld.bat

```diff
@@ -12,6 +12,9 @@ cmake                                                                        ^
     -DENABLE_SYSTEM_OPENCL=ON                                                ^
     -DENABLE_SYSTEM_PROTOBUF=ON                                              ^
     -DENABLE_SYSTEM_SNAPPY=ON                                                ^
+    -DOPENVINO_EXTRA_MODULES="%SRC_DIR%/openvino_contrib/modules"            ^
```

Comment:
you can explicitly specify `-DOPENVINO_EXTRA_MODULES="%SRC_DIR%/openvino_contrib/modules/nvidia_plugin"`